server/routes/appointments/appointmentsController.ts

```diff
@@ -1122,6 +1235,61 @@ export default class AppointmentsController {
     return this.interventionsService.getActionPlanAppointment(accessToken, actionPlanId, Number(sessionNumber))
   }
 
+  private extractSupplierAssessmentAppointment(
+    fetchResult: DraftFetchSuccessResult<DraftAppointmentBooking | DraftAppointment>,
+    id: string
+  ): InitialAssessmentAppointment | null {
+    const { draft } = fetchResult
+    const appointmentSchedulingDetails = this.extractAppointmentSchedulingDetails(draft)
+    if (appointmentSchedulingDetails) {
+      const draftAppointment = draft.data as DraftAppointment
+      if (draftAppointment?.sessionFeedback) {
+        return {
+          sessionFeedback: draftAppointment.sessionFeedback,
+          id,
+          ...appointmentSchedulingDetails,
+        }
+      }
+      return {
```

Comment:
i think we could use a comment on why we have to construct this empty object here. (especially because it's never used)